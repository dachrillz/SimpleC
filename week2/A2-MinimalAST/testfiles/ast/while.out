Program
  List
    Function ID="a"
      null
      List
        While
          Plus
            Var ID="5"
            Var ID="5"
          List
            IdUse ID="a"
              Var ID="5"
        Return
          null
