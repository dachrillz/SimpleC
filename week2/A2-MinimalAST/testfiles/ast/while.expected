Program
  List
    Function ID="a"
      null
      List
        While
          Plus
            Numeral NUMERAL="5"
            Numeral NUMERAL="5"
          List
            IdUse ID="a"
              Numeral NUMERAL="5"
        Return
